src/main/java/org/tron/core/db/Manager.java

```diff
@@ -1324,6 +1345,16 @@ public void processBlock(BlockCapsule block)
     //reset BlockEnergyUsage
     this.dynamicPropertiesStore.saveBlockEnergyUsage(0);
 
+    //parallel check sign
+    if (!block.generatedByMyself) {
+      try {
+        preValidateTransactionSign(block);
+      } catch (InterruptedException e) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1917&issues=AWgnd9jpUMkE2J58yg77&open=AWgnd9jpUMkE2J58yg77)

[ISSUE_KEY]: AWgnd9jpUMkE2J58yg77